Library/Homebrew/cmd/untap.rb

```diff
@@ -5,7 +5,7 @@ def untap
     raise "Usage is `brew untap <tap-name>`" if ARGV.empty?
 
     ARGV.each do |tapname|
-      user, repo = tap_args(tapname)
+      user, repo = tap_args()
```

Comment:
This changes existing behaviors as it used to be able to untap multiple taps simultaneously.